src/AppBundle/Controller/BlogController.php

```diff
@@ -34,12 +34,20 @@
 class BlogController extends Controller
 {
     /**
-     * @Route("/", name="blog_index")
+     * @Route("/", name="blog_index", defaults={"page" = 1})
+     * @Route("/page/{page}", name="blog_index_paginated", defaults={"page" = 1}, requirements={"page" : "\d+"})
```

Comment:
Why do you add an default for page?

/page/ is not a valid path for me, I would remove the default for page on the "blog_index_paginated"